docker/Dockerfile-test-scripts

```diff
@@ -0,0 +1,30 @@
+# test e.g. via
+#   docker build --pull -t install_scripts:test -f docker/Dockerfile-test-scripts .
```

Comment:
Sure, fine, but could we just maybe have some kind of identifier prefix? E.g. sth like one of the two below
```suggestion
#   docker build --pull -t smarp_install_scripts:test -f docker/Dockerfile-test-scripts .
#   docker build --pull -t pandoc_tinytex:test -f docker/Dockerfile-test-scripts .
```

Reply:
discussed and agreed